global.json

```diff
@@ -1,16 +1,16 @@
 {
   "sdk": {
-    "version": "8.0.100-preview.7.23376.3",
+    "version": "8.0.100-rc.1.23455.8",
```

Comment:
@lewing Should this be rtm? Also the one below.

Reply:
Just asking because we've been doing similar manual adjustments in wasm flows. I don't know if it applies to this as well.

Reply:
Not yet, we have to wait until we have an RTM SDK.  I believe the arcade insertion is just doing it's thing propagating the arcade toolset changes here.

Reply:
We'll need a newer sdk to be able to take the new roslyn in https://github.com/dotnet/runtime/pull/92503 but this pr is just to catch us up with normal arcade flow